src/types.ts

```diff
@@ -21,7 +21,10 @@ interface BaseWebhookEvent<TName extends WebhookEventName> {
   payload: WebhookEventMap[TName];
 }
 
-export interface Options<TTransformed = unknown> {
+export interface Options<
+  E extends EmitterWebhookEventName = any,
+  TTransformed extends EmitterWebhookEvent<E> = any
```

Comment:
I would like to improve these `any` default values here. Is it possible?

Reply:
You can default to `EmitterWebhookEventName`, it's done elsewhere in the code